### PR TITLE
Installing on IBM Z updates

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@bergerhoffer This PR contains the updates for IBM Z. There's no 4.3-specific text here, and the IBM Power assemblies and entries in the _topic_map.yaml have been removed. The IBM Power conditionals remain.

**Review by bergerhoffer — COMMENTED** [review]
Just a few things!

File: modules/installation-approve-csrs.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ these CSRs are approved or, if necessary, approve them yourself.
 +
 ifdef::ibm-z[]
 ----
-# oc get nodes
+$ oc get nodes
```
[the remark this answers is not in the document]
Comment (rh-max) [reply modules/installation-approve-csrs.adoc:36]: Actually, made it just to comply with the other part of the conditional. That aside, `$` is safer anyway.

File: modules/installation-installing-bare-metal.adoc [hunk 2]
```diff
@@ -38,10 +38,10 @@ endif::restricted[]
 ----
 $ ./openshift-install --dir=<installation_directory> wait-for bootstrap-complete \ <1>
     --log-level=info <2>
-
-INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
-INFO API v1.14.6+c4799753c up
-INFO Waiting up to 30m0s for the bootstrap-complete event...
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com...
+INFO API v1.16.2 up
```
[the remark this answers is not in the document]
Comment (rh-max) [reply modules/installation-installing-bare-metal.adoc:42]: Good catch, changed the whole output.

File: modules/installation-approve-csrs.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/installation-approve-csrs.adoc:36]: Sure that's fine. If that's the case and it's still true for 4.3, you might want to make the change there too

File: modules/installation-bare-metal-config-yaml.adoc [hunk 3]
```diff
@@ -102,20 +119,24 @@ load balancers and routers to manage the traffic.
...
 endif::restricted[]
 ifdef::restricted[]
-<11> For `bastion_host_name`, specify the registry domain name
+<12> For `bastion_host_name`, specify the registry domain name
 that you specified in the certificate for your mirror registry, and for
 `<credentials>`, specify the base64-encoded user name and password for
 your mirror registry.
 endif::restricted[]
-<12> The public portion of the default SSH key for the `core` user in
+<13> The public portion of the default SSH key for the `core` user in
```
[the remark this answers is not in the document]
Comment (rh-max) [reply modules/installation-bare-metal-config-yaml.adoc]: Fixed.

**Comment by bergerhoffer** [comment]
This LGTM now!

**Pull request merged**